### PR TITLE
Updated Header to Match Current UI Wireframe

### PR DESCRIPTION
**Pull request by vuonga1103**
- Created new routes and stub components
- Updated DEFAULT_COMMON_MENU object
- Updated common-menu collection in MongoDB
Screen Shot 2020-12-20 at 6 56 55 PM
Screen Shot 2020-12-20 at 6 57 12 PM
Screen Shot 2020-12-20 at 6 57 25 PM
Screen Shot 2020-12-20 at 6 57 35 PM

### CODE REVIEW

File: src/components/Header/Header.js [hunk 1]
```diff
@@ -241,13 +241,12 @@ function Header() {
...
           ({ href }) =>
             href !== '/' &&
-            href !== '/register' &&
             href !== '/notifications' &&
             href !== '/login?logout=true' &&
             href !== '/settings'
         )
         .map(({ id, href, label }) => {
-          let color = label === 'Donate' ? '#B9EBEC' : 'white';
+          let color = label === 'Join Us' ? '#B9EBEC' : 'white';
```
Comment (cfernando415) [inline src/components/Header/Header.js:249]: make it a const if you're not modifying the value anywhere else

**Review by cfernando415 — CHANGES_REQUESTED** [review]
please see comment

File: src/components/Header/Header.js (hunk 1 above)
Comment (vuonga1103) [reply src/components/Header/Header.js]: resolved in 750b857

**Review by cfernando415 — APPROVED** [review]
:+1:

**Pull request merged**